### PR TITLE
add support for hi, gu, de, ja, ru

### PR DESCRIPTION
**Pull request by MaxMillington**
Add support for hindi, gujarati, german, russian, japanese

### CODE REVIEW

File: packages/web/localizations/es.json [hunk 1]
```diff
@@ -910,4 +910,4 @@
     }
   },
   "disclaimer": "Descargo de responsabilidad"
-}
+}
```
Comment (jonator) [inline packages/web/localizations/es.json:913]: I think not ending in line breaks will fail the linter IIRC

Comment (MaxMillington) [reply packages/web/localizations/es.json]: fixed

**Comment by DavideSegullo** [comment]
@MaxMillington tests are failing

**Review by DavideSegullo — CHANGES_REQUESTED** [review]
Options are missing inside settings modal, we need to list them [here](https://github.com/osmosis-labs/osmosis-frontend/blob/add-languages/packages/web/stores/user-settings/language.tsx#L13)

**Comment by DavideSegullo** [comment]
> Options are missing inside settings modal, we need to list them [here](https://github.com/osmosis-labs/osmosis-frontend/blob/add-languages/packages/web/stores/user-settings/language.tsx#L13)

Also the dialog is buggy when we have a large list of options:

Screenshot 2024-01-16 alle 19 23 47

**Comment by MaxMillington** [comment]
> Options are missing inside settings modal, we need to list them [here](https://github.com/osmosis-labs/osmosis-frontend/blob/add-languages/packages/web/stores/user-settings/language.tsx#L13)

good catch

**Comment by MaxMillington** [comment]
> > Options are missing inside settings modal, we need to list them [here](https://github.com/osmosis-labs/osmosis-frontend/blob/add-languages/packages/web/stores/user-settings/language.tsx#L13)
> 
> Also the dialog is buggy when we have a large list of options:
> 
> Screenshot 2024-01-16 alle 19 23 47

what's your zoom % @DavideSegullo? I can only recreate if I zoom in a bunch. At 100%, it looks fine for me.

Screenshot 2024-01-17 at 9 53 01 AM

**Comment by DavideSegullo** [comment]
> > > Options are missing inside settings modal, we need to list them [here](https://github.com/osmosis-labs/osmosis-frontend/blob/add-languages/packages/web/stores/user-settings/language.tsx#L13)
> > 
> > 
> > Also the dialog is buggy when we have a large list of options:
> > Screenshot 2024-01-16 alle 19 23 47
> 
> what's your zoom % @DavideSegullo? I can only recreate if I zoom in a bunch. At 100%, it looks fine for me.
> 
> Screenshot 2024-01-17 at 9 53 01 AM

Yes zoom is 100%, it happens when you scroll down at the end of the list, also my screen is 1440x800

**Comment by kamal-sutra** [comment]
Selected Hindi, but nothing changed
image
image

**Comment by kamal-sutra** [comment]
Also buggy overlap as mentioned by Davide
image

**Comment by CryptoAssassin1** [comment]
Newly added languages are not translating.

**Comment by DavideSegullo** [comment]
We also need to add banner translations:

Screenshot 2024-01-18 alle 19 26 11

**Comment by MaxMillington** [comment]
> We also need to add banner translations:
> 
> Screenshot 2024-01-18 alle 19 26 11

Way ahead of you https://github.com/osmosis-labs/fe-content/pull/11

**Review by DavideSegullo — APPROVED** [review]

**Pull request merged**